Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/system/classes/EditClassLayout.java

```diff
@@ -131,14 +133,14 @@ public void execute(Boolean value) {
      *
      * @param name Class name
      * @param groups Class groups
-     * @param engine Class engine
+     * @param engines Class engines
      */
-    public void setClass(String name, String groups, String engine) {
+    public void setClass(String name, String groups, String engines) {
 
         if (name != null) {
             this.nameField.setValue(name);
             this.nameField.setDisabled(true);
-            this.enginesPickList.setValue(engine);
+            this.enginesPickList.setValues(engines.split(", "));
```

Comment:
Wow, thanks for the suggestion, I wasn't aware of these new functionalities.:-) And it actually goes a bit further, since the Stream<String> needs to be converted to a String [] : 
this.enginesPickList.setValues(Arrays.asList(engines.split(",")).stream().map(String::trim).toArray(elem -> new String[elem]));

Just a note on my previous message, which was "misinterpreted" by the browser. The argument between " " was actually \\s*,\\s* , so it should have worked fine with and without empty spaces. 

Your suggestion is mush nicer though, so I'll be happy to implement it.:-)